src/sentry/utils/snuba.py

```diff
@@ -591,6 +593,8 @@ def _prepare_start_end(
         start = datetime(2008, 5, 8)
     if not end:
         end = datetime.utcnow() + timedelta(seconds=1)
+    assert isinstance(start, datetime)
+    assert isinstance(end, datetime)
```

Comment:
I don't think these asserts should be necessary -- mypy should know there's a narrowing due to the nullability checks and assignments here

what error did this fix?

Reply:
yeah it was unnecessary,  not sure why mypy was showing "can be datetime | None" before